joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 100

left_joint_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: left/wheel_fixed_joint
  pid: {p: 1,i: 0,d: 0,i_clamp_min: -10,i_clamp_max: 10}
  # pid: {p: 0.001, i: 0.0, d: 0.0}

front_joint_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: front/wheel_fixed_joint
  # pid: {p: 0.001, i: 0.0, d: 0.0}
  pid: {p: 1,i: 0,d: 0,i_clamp_min: -10,i_clamp_max: 10}

right_joint_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: right/wheel_fixed_joint
  pid: {p: 1,i: 0,d: 0,i_clamp_min: -10,i_clamp_max: 10}
  # pid: {p: 0.001, i: 0.0, d: 0.0}
# gazebo_ros_control:
#   pid_gains:
#     left/wheel_fixed_joint:
#       p: 100.0
#       i: 0.0
#       d: 0.0
#     front/wheel_fixed_joint:
#       p: 100.0
#       i: 0.0
#       d: 0.0
#     right/wheel_fixed_joint:
#       p: 100.0
#       i: 0.0
#       d: 0.0


